components/modal/edit-expiry-modal.tsx

```diff
@@ -33,11 +32,7 @@ const EditExpiryModal = ({
       </div>
       <div className="flex flex-col relative bg-white dark:bg-gray-800 rounded-lg p-4 mb-4">
         <div className="px-6 mb-2">
-          <Slider
-            length={MAX_EPOCHS}
-            value={selected}
-            onValueChange={setSelected}
-          />
+          <Slider length={4} value={selected} onValueChange={setSelected} />
```

Comment:
```suggestion
          <Slider length={MAX_EPOCHS} value={selected} onValueChange={setSelected} />
```

Reply:
This is temporary code. Since MAX_EPOCHS is not needed in the future, there is no need to create it.